app/game/typingGame/script.js

```diff
@@ -0,0 +1,56 @@
+//Maybe replace spaceQuotes with random quote API?
+const spaceQuotes = [
+    "The question of whether a computer can think is no more interesting than the question of whether a submarine can swim.",
+    "Sometimes it seems as though each new step towards AI, rather than producing something which everyone agrees is real intelligence, merely reveals what real intelligence is not.",
+    "People worry that computers will get too smart and take over the world, but the real problem is that they're too stupid and they've already taken over the world.",
+    "By far the greatest danger of Artificial Intelligence is that people conclude too early that they understand it.",
+    "The AI does not hate you, nor does it love you, but you are made out of atoms which it can use for something else.",
+    "With few ambitions, most people allowed efficient machines to perform everyday tasks for them. Gradually, humans ceased to think, or dream... or truly live."
+]
+const quoteDisplay = document.getElementById("quoteDisplay");
+const quoteInput = document.getElementById("quoteInput");
+const messageDisplay = document.getElementById("messageDisplay");
+
+quoteInput.addEventListener("input", () => {
```

Comment:
Clever, you're checking the value of the input field rather than manually adding the logic to see what the user is typing. This leverages the existing functionality in the input field which frees us from managing that functionality ourselves. Excellent work!